content/id/docs/concepts/policy/pod-security-policy.md

```diff
@@ -0,0 +1,469 @@
+
+```shell
+kubectl-user get pods --watch
+NAME                    READY     STATUS    RESTARTS   AGE
+pause-7774d79b5-qrgcb   0/1       Pending   0         1s
+pause-7774d79b5-qrgcb   0/1       Pending   0         1s
+pause-7774d79b5-qrgcb   0/1       ContainerCreating   0         1s
+pause-7774d79b5-qrgcb   1/1       Running   0         2s
+```
+
+### Membersihkan
+
+Hapus Namespace tersebut untuk membersihkan sebagian besar sumber daya yang digunakan dalam contoh ini:
+
+```shell
+kubectl-admin delete ns psp-example
+namespace "psp-example" deleted
+```
+
+Perlu diperhatikan bahwa sumber daya `PodSecurityPolicy` tidak diberi Namespace, dan harus dibersihkan secara terpisah:
+
+```shell
+kubectl-admin delete psp example
+podsecuritypolicy "example" deleted
+```
+
+### Contoh-contoh Kebijakan
+
+Berikut adalah kebijakan dengan batasan paling sedikit yang dapat kamu buat, ekuivalen dengan tidak menggunakan _admission controller_ Pod Security Policy:
+
+{{< codenew file="policy/privileged-psp.yaml" >}}
+
+Berikut adalah sebuah contoh kebijakan yang restriktif yang mengharuskan pengguna-pengguna untuk berjalan sebagai pengguna yang _unprivileged_, memblokir kemungkinan eskalasi menjadi _root_, dan mengharuskan penggunaan beberapa mekanisme keamanan.
+
+{{< codenew file="policy/restricted-psp.yaml" >}}
+
+## Referensi Kebijakan
+
+### Privileged
+
+**Privileged** - menentukan bila Container manapun di dalam sebuah Pod dapat mengaktifkan mode _privileged_. Secara bawaan, sebuah Container tidak diizinkan untuk mengakses perangkat apapun pada _host_-nya, tapi sebuah Container yang "_privileged_" akan diberikan akses untuk semua perangkat pada _host_-nya. Hal ini mengizinkan hampir semua akses yang sama dengan proses yang berjalan pada _host_ kepada Container tersebut. Hal ini berfungsi untuk Container-container yang ingin menggunakan kemampuan Linux seperti memanipulasi _network stack_ atau mengakses perangkat-perangkat.
+determines if any container in a pod can enable privileged mode.
+
+### Namespace Host
+
+**HostPID** - Mengatur jika Container-container pada Pod dapat berbagi _namespace process ID_ pada _host_. Catatlah bahwa saat dipasangkan dengan ptrace, hal ini dapat digunakan untuk eskalasi _privilege_ di luar kontainer (ptrace secara bawaan tidak diizinkan).
+
+**HostIPC** - Mengatur jika Container-container pada Pod dapat berbagi _namespace IPC_ pada _host_.
+
+**HostNetwork** - Mengatur jika Pod dapat menggunakan _namespace_ jaringan pada _host_. Melakukan hal ini akan memberikan Pod akses pada perangkat _loopback_, _service_ yang sedang _listening_ pada _localhost_, dan dapat digunakan untuk mengintai aktivitas jaringan pada Pod-pod lain pada Node yang sama.
+
+**HostPorts** - Memberikan daftar putih dari berbagai _port_ yang diizinkan pada _namespace_ jaringan pada _host_. Hal ini didefinisikan sebagai sebuah daftar `HostPortRange`, dengan `min`(inklusif) dan `max`(inklusif). Nilai bawaannya adalah tidak ada _host port_ yang diizinkan.
```

Comment:
will we use daftar putih or white list for the next term that will be appeared in our translation?

Reply:
I think we can translate it. It practically refers to the "list of allowed thing".
.
.
But, kbbi says a different thing, though it kinda has a similar meaning:
https://kbbi.kemdikbud.go.id/entri/daftar%20putih
https://kbbi.kemdikbud.go.id/entri/daftar%20hitam

@irvifa @girikuncoro WDYT?